src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -38,11 +40,11 @@ INT NPrecedenceOfOp(int nopCode)
     int iPrec;
 
     iPrec = 0;
-    while ((iPrec < ARRAYSIZE(rgbPrec)) && (nopCode != rgbPrec[iPrec]))
+    while ((iPrec < std::size(rgbPrec)) && (nopCode != rgbPrec[iPrec]))
```

Comment:
Add `using namespace std;` for cpp files and drop the fully qualified name when standard functions.